inspire_dojson/utils/__init__.py

```diff
@@ -130,7 +130,7 @@ def get_record_ref(recid, endpoint='record'):
 
 def legacy_export_as_marc(json, tabsize=4):
     """Create the MARCXML representation using the producer rules."""
-    export = ['<record>\n']
+    export = [u'<record>\n']
```

Comment:
since there are so many occurrences on this file, it would be cleaner to add:

```python
from future import unicode_literals
```

Reply:
If the error that @kaplun was trying to solve was [this](https://travis-ci.org/inspirehep/inspire-dojson/jobs/308514082#L680), this is not the right solution...

Reply:
Yup.

Reply:
That error was just saying that the interpolation string "Conversion from JSON to MARCXML..." needed to be a unicode object, as otherwise the usual implicit ASCII conversion happens when you interpolate inside the schema name.

So all you needed was to add a `u` in front of that string : )